central/networkgraph/flow/datastore/internal/store/testcommon/flow.go

```diff
@@ -172,6 +172,11 @@ func (suite *FlowStoreTestSuite) TestStore() {
 func (suite *FlowStoreTestSuite) TestRemoveAllMatching() {
 	t1 := time.Now().Add(-5 * time.Minute)
 	t2 := time.Now()
+	if features.PostgresDatastore.Enabled() {
+		// Round the timestamps to the microsecond
+		t1 = time.Date(t1.Year(), t1.Month(), t1.Day(), t1.Hour(), t1.Minute(), t1.Second(), (t1.Nanosecond()/1000)*1000, t1.Location())
```

Comment:
nitpick: I'm not sure if we can use `t1 = t1.Truncate()` here. https://pkg.go.dev/time#Time.Truncate